README.md

```diff
@@ -8,7 +8,11 @@ are available in the Amazon Elastic Container Registry (Amazon ECR).
 
 The AWS DLCs are used in Amazon SageMaker as the default vehicles for your SageMaker jobs such as training, inference, 
 transforms etc. They've been tested for machine
```

Comment:
"such as training, inference, or transforms. All AWS DLCs are tested for machine learning..."
+ Remove etc.
+ Change passive "they have" to active "are"